README.md

```diff
@@ -27,3 +28,10 @@ enables [debug mode](#debug-mode).
 ## Enable trace support
 
 See [the tracing guide](TRACING.md).
+
+## Enable debug console
+
+Add `init=/usr/bin/kata-agent` and `agent.debug_console` to the guest kernel command line to
```

Comment:
Why is the `init=` required? Doesn't that imply this only works for initrd images? But if so, below you mention that you can use a rootfs or an initrd type image?

Reply:
oops!, `init=` is not required and yes, debug console works for rootfs and initrd images :smile:

Reply:
fixed, thanks